tests/model_registry/rest_api/test_model_registry_secure_db.py

```diff
@@ -0,0 +1,131 @@
+import pytest
+import requests
+from typing import Self
+import copy
+from pytest_testconfig import config as py_config
+from tests.model_registry.rest_api.utils import register_model_rest_api, validate_resource_attributes
+from tests.model_registry.rest_api.constants import MODEL_REGISTER_DATA
+from tests.model_registry.utils import generate_random_name
+from tests.model_registry.constants import CA_MOUNT_PATH, SECURE_MR_NAME
+from tests.model_registry.utils import get_model_registry_endpoint
+from kubernetes.dynamic import DynamicClient
+from utilities.constants import DscComponents
+
+from simple_logger.logger import get_logger
+
+
+LOGGER = get_logger(name=__name__)
+
+
+@pytest.mark.parametrize(
+    "updated_dsc_component_state_scope_class",
```

Comment:
For main we must use oauth.

Reply:
if it is by default and not secure_db related it should not be addressed in this change to avoid scope creep

Reply:
With Luca's change merged, this comment is no longer valid. Resolving it.